src/pages/en/reference/directives-reference.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Directives Reference
+---
+
+And UI Framework components. Directives can only be used inside of Astro components.
```

Comment:
Did you mean to leave this in? It seems like a fragment from the previous draft.

Reply:
Whoops, yep